modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -52,13 +52,15 @@ kind: Kiali
 metadata:
   name: kiali-user-workload-monitoring
   namespace: istio-system
-spec:
-  external_services:
-    istio:
-      url_service_version: 'http://istiod-basic.istio-system:15014/version'
-      config_map_name: istio-basic # <1>
-    prometheus:
-      auth:
+spec: 
+  external_services: 
+    istio: 
+      config_map_name: istio-<smcp-name>
+      istio_sidecar_injector_config_map_name: istio-sidecar-injector-<smcp-name>
+      istiod_deployment_name: istiod-<smcp-name>
+      url_service_version: 'http://istiod-<smcp-name>.istio-system:15014/version'
```

Comment:
Two questions:
1) Should the URL have the single quotes around it? (the one below does not)
2) Should the URL above start with https, like the one below?

Reply:
1. It might have, but it's not necessary
2. No.